build/repo.targets

```diff
@@ -1,7 +1,8 @@
 <Project>
   <Import Project="VSIX.targets" />
   <ItemGroup>
-    <Solutions Update="..\Razor.sln">
+    <!-- It's important to use RepositoryRoot in the file path, not relative paths. Relative paths are relative to the build context of KoreBuild, not this file. -->
```

Comment:
No need for the comment, we shouldn't have been using a relative path here to begin with.

Reply:
ok.